qase-javascript-commons/changelog.md

```diff
@@ -1,3 +1,10 @@
+# qase-javascript-commons@2.0.0-beta.5
+
+## What's new
+
+* Update the config of reporters. Added `captureLogs` field. If it is set to `true`, the reporter will capture logs from the test framework.
```

Comment:
It's not quite clear where this field should be, because the config is a multi-level data structure. Could you give an example?

Reply:
And there's an env variable too.

Reply:
Oh, I've found it below